src/BEditaClient.php

```diff
@@ -314,13 +314,21 @@ public function removeRelated($id, string $type, string $relation, array $data,
      * @param string $relation Relation name
      * @param string $data Related resources or objects to insert
      * @param array|null $headers Custom request headers
+     * @param bool|null $meta Should post relationships metadata.
      * @return array|null Response in array format
      */
-    public function replaceRelated($id, string $type, string $relation, array $data, ?array $headers = null): ?array
+    public function replaceRelated($id, string $type, string $relation, array $data, ?array $headers = null, ?bool $meta = false): ?array
```

Comment:
Do we need a `$meta` argument here? 
Could we simply check `$body` and do a POST if meta are set?

Reply:
I thought that introducing the `$meta` argument would be a safer solution for compatibility, but I agree that checking the `$body` would be the better thing in general. Up to you the choice :)

Reply:
after quick check with @batopa the best solution seems:
* arguments list stay unchanged
* after PATCH extract from `$body` an array of items having `meta` 
* if this array is not empty => perform a POST with this subset as `$body`